### PR TITLE
Sorting Providers within Sandbox

### PR DESCRIPTION
**Pull request by lgarceau768**
## Change Details
Adding the ability to sort private keys and client tokens by the various options (name, creation date, expiration date) (ascending and descending). Just modifications to the front end of the application to only sort them via the client side. No interaction with the database is needed. 

Needed to modify the `accordian.js` in order to fix a bug that occured where the user could not open the delete button within a sorted table.

## Security Implications
- [x] new software dependencies. 

USWDS CDN (added in the `_head.html.erb`)
```
Init Script for USWDS https://cdnjs.cloudflare.com/ajax/libs/uswds/2.13.3/js/uswds-init.min.js
Stylesheet for USWDS https://cdnjs.cloudflare.com/ajax/libs/uswds/2.13.3/css/uswds.min.css
```

USWDS CDN (added in the `_footer.html.erb`)
```
Main Script for USWDS
https://cdnjs.cloudflare.com/ajax/libs/uswds/2.13.3/js/uswds.min.js
```
I added `bootstrap-table-rails` plugin to the `GemFile` [(link to source)](https://github.com/bjevanchiu/bootstrap-table-rails). Used for the sorting logic of the table

Does not have any security implications (purely front end sorting) and use of the table. 
The existing architecture was modified from using `li, ul, and div` components to now use accessible `table, thead, td, and tr` elements. These changes occurred in `_assigned_content.html.erb`

### CODE REVIEW

File: dpc-web/app/assets/stylesheets/components/_accordions.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
   list-style-type: none;
   margin: 0;
   padding: 0;
-  width: 100%;
+  width: 17.5rem;
```
Comment (lgarceau768) [inline dpc-web/app/assets/stylesheets/components/_accordions.scss:5]: This change was needed in order to preserve the correct sizing of the modal delete box as it opened and closed. I did not want a large resize of the table to occur.

File: dpc-web/app/controllers/portal_controller.rb [hunk 2]
```diff
@@ -5,5 +5,10 @@ class PortalController < ApplicationController
 
   def show
     @user = current_user
+    if !current_user.unassigned?
+      @client_tokens = @user.primary_organization.reg_org.client_tokens()
+    else
+      @client_tokens = []
+    end
   end
```
Comment (lgarceau768) [inline dpc-web/app/controllers/portal_controller.rb:13]: The logic behind this file is that I now want to be able to manipulate the list of `@client_tokens` within the controller file so it is later reflected within the table and front end

File: dpc-web/app/assets/javascripts/application.js [hunk 3]
```diff
@@ -21,5 +21,5 @@
 //= require activestorage
 //= require_tree ./utils
 //= require_tree .
-
+//= require bootstrap-table/bootstrap-table
```
Comment (lgarceau768) [inline dpc-web/app/assets/javascripts/application.js:24]: Needed by bootstrap in order to bundle correctly with rails

File: dpc-web/app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -51,4 +51,4 @@
  @import "components/public_keys";
  @import "components/radio_buttons";
  @import "components/site-blocks";
- @import "utilities/add-ons";
+ @import "utilities/add-ons";
```
Comment (lgarceau768) [inline dpc-web/app/assets/stylesheets/application.scss:54]: I think this may be something like the files line ending (LF/CRLF)

**Review by mftcodes — APPROVED** [review]
This looks good to me. I was playing around with some of this as well when I was looking at paging. It appears to work well.

**Review by TiannaW24 — APPROVED** [review]
LGTM

**Pull request merged**